### PR TITLE
Feature/mock message page

### PR DESCRIPTION
**Pull request by tchanee**
Hey guys! I worked on a mock message page, more specifically a view that shows you your conversations with other users.

As of now it is made up of clickable dummy conversations that will have to be replaced later on with actual data from our database.

### CODE REVIEW

File: app/src/androidTest/java/ch/proximeety/proximeety/presentation/views/conversationList/ConversationListViewTest.kt [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    val hiltRule = HiltAndroidRule(this)
+
+    @get:Rule(order = 1)
+    val composeTestRule = createAndroidComposeRule<MainActivity>()
+
+    @get:Rule
+    val instantTaskExecutorRule = InstantTaskExecutorRule()
+
+    @Inject
+    lateinit var userInteractions : UserInteractions
```
Comment (yanisdb) [inline app/src/androidTest/java/ch/proximeety/proximeety/presentation/views/conversationList/ConversationListViewTest.kt:33]: You can remove this line.

File: app/src/androidTest/java/ch/proximeety/proximeety/presentation/views/conversationList/ConversationListViewTest.kt [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+class ConversationListViewTest {
+
+    @get:Rule(order = 0)
+    val hiltRule = HiltAndroidRule(this)
+
+    @get:Rule(order = 1)
+    val composeTestRule = createAndroidComposeRule<MainActivity>()
+
+    @get:Rule
+    val instantTaskExecutorRule = InstantTaskExecutorRule()
```
Comment (yanisdb) [inline app/src/androidTest/java/ch/proximeety/proximeety/presentation/views/conversationList/ConversationListViewTest.kt:30]: You can remove this line

File: app/src/main/java/ch/proximeety/proximeety/presentation/navigation/graphs/AuthenticationNavigationGraph.kt [hunk 3]
```diff
@@ -18,5 +18,6 @@ fun NavGraphBuilder.authenticationNavigationGraph() {
         ) {
             AuthenticationHomeView()
         }
+
```
Comment (yanisdb) [inline app/src/main/java/ch/proximeety/proximeety/presentation/navigation/graphs/AuthenticationNavigationGraph.kt:21]: Empty line here

**Review by yanisdb — DISMISSED** [review]
Composable functions should start with uppercase

File: app/src/androidTest/java/ch/proximeety/proximeety/presentation/views/conversationList/ConversationListViewTest.kt (hunk 1 above)
Comment (tchanee) [reply app/src/androidTest/java/ch/proximeety/proximeety/presentation/views/conversationList/ConversationListViewTest.kt:33]: you mean from line 30 to 33 right?

File: app/src/main/java/ch/proximeety/proximeety/presentation/views/conversationList/ConversationListModel.kt [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    val senderImage: String? = null
+)
+
+var msg1 = MessagesModel("333", "test1", "sender1")
+
+val msg2 = MessagesModel(
+    sender = "Test sender",
+    time = "10:00 AM",
+    message = "Hello there, how are you?"
+)
```
Comment (Hamzezi) [inline app/src/main/java/ch/proximeety/proximeety/presentation/views/conversationList/ConversationListModel.kt:19]: add new empty line at the end of the file

File: app/src/main/java/ch/proximeety/proximeety/presentation/views/conversationList/ConversationListUIEvent.kt [hunk 5]
```diff
@@ -0,0 +1,4 @@
+package ch.proximeety.proximeety.presentation.views.conversationList
+
+class ConversationListUIEvent {
+}
```
Comment (Hamzezi) [inline app/src/main/java/ch/proximeety/proximeety/presentation/views/conversationList/ConversationListUIEvent.kt:4]: add a new empty line at the end of the file

File: app/src/main/java/ch/proximeety/proximeety/presentation/views/conversationList/ConversationListViewModel.kt [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+    fun onEvent(event: ConversationListEvent) {
+        when (event) {
+            is ConversationListEvent.ConversationClick -> {
+                navigationManager.navigate(MainNavigationCommands.home)
+            }
+        }
+    }
+
+}
```
Comment (Hamzezi) [inline app/src/main/java/ch/proximeety/proximeety/presentation/views/conversationList/ConversationListViewModel.kt:32]: add a new empty line at the end of the file

File: app/src/main/java/ch/proximeety/proximeety/presentation/views/conversationList/ConversationListView.kt [hunk 7]
```diff
@@ -0,0 +1,197 @@
...
+//    ) {
+//        Icon(
+//            imageVector = Icons.Default.Person,
+//            contentDescription = "",
+//            tint = Color.LightGray,
+//            modifier = Modifier.fillMaxSize()
+//        )
+//    }
+//}
+//
```
Comment (Hamzezi) [inline app/src/main/java/ch/proximeety/proximeety/presentation/views/conversationList/ConversationListView.kt:197]: idk if the commented code was left intentionally or not, so I thought I'd mention it too

**Review by Hamzezi — APPROVED** [review]
I commented on some minor details that you can fix on your next PR, but other than that it looks good to me, nice work.

**Review by belengg27 — APPROVED** [review]
Good work

**Pull request merged**